src/output.h

```diff
@@ -1,14 +1,14 @@
 #pragma once
 
 #include "config.h"
+#include "frame.h"
```

Comment:
I am guessing this is here to pull in `MAX_AUDIO_PACKETS`. If so, it may probably be better to move it to `defines.h`, which has constants that are needed in more than one file.

Reply:
Most likely a good idea + I don't think MAX_AUDIO_PACKETS constant needs to be publicly accessible outside the library.